app/Http/Controllers/UserController.php

```diff
@@ -10,41 +10,42 @@ class UserController extends Controller
 {
     public function login()
     {
-        return view('login');
+        return view('Login.login');
```

Comment:
```suggestion
        return view('login.login');
```
tên file view ko nên viết hoa nhé